Sources/Tracking/Service/HttpClient.swift

```diff
@@ -127,4 +128,15 @@ extension CIOHttpClient {
 
         return encodedRawHeader.base64EncodedString(options: NSData.Base64EncodingOptions(rawValue: 0))
     }
+    
+    static func getUserAgent() -> String {
+        var userAgent = "Customer.io iOS Client/"
+        userAgent += SdkVersion.version
+        #if canImport(UIKit)
+        userAgent += " (\(DeviceInfo.deviceInfo); \(DeviceInfo.osInfo))"
+        userAgent += " \(DeviceInfo.customerAppName)/"
+        userAgent += DeviceInfo.customerAppVersion
+        #endif
+        return userAgent
```

Comment:
I think it would be helpful to add a code comment for this function with the User-Agent format the API requires. It would be good documentation for the iOS dev team to make sure we are consistent with the requirement.

Reply:
Added detailed comment on this function along with the expected results when UIKit is found and when not found